docs/intellicode-visual-studio-code.md

```diff
@@ -67,15 +67,14 @@ If you see IntelliSense completion lists, but none of the completion items are s
    As a first step, try updating to Visual Studio Code 1.30 or later, and [turning on network proxy support](https://code.visualstudio.com/updates/v1_30#_network-proxy-support-for-extensions) .
```

Comment:
It's 1.3 here but 1.58 just a few lines above. Was that intentional?

Reply:
mb - fixed